src/components/markdown-help.jsx

```diff
@@ -0,0 +1,269 @@
+            </td>
+          </tr>
+          <tr>
+            <td>Header</td>
+            <td>
+              # header1<br />
+              ## header2<br />
+              ### header3<br />
+              <em>etc., up to six # symbols</em>
+            </td>
+            <td>
+              <Markdown>{headers}</Markdown>
+            </td>
+          </tr>
+          <tr>
+            <td>Code Blocks</td>
+            <td>
+              <span>`code block`</span><br />
+              <span><em>or with four (4) spaces before (and optionally, after)</em></span><br />
+              <span>&nbsp;&nbsp;&nbsp;&nbsp;code block&nbsp;&nbsp;&nbsp;&nbsp;</span>
+            </td>
+            <td>
+              <Markdown>`code block`</Markdown>
+            </td>
+          </tr>
+          <tr>
+            <td>Strikethrough</td>
+            <td>~~strikethrough~~</td>
+            <td><Markdown>~~strikethrough~~</Markdown></td>
+          </tr>
+          <tr>
+            <td>Image</td>
+            <td>
+              ![imagealttext](/assets/simple-avatar.png)<br />
+              <em>images must already be uploaded; use <a href="http://imgur.com/" rel="noopener noreferrer" target="_blank">imgur</a> to host new images</em>
+            </td>
+            <td>
+              <Markdown>{avatarImage}</Markdown>
+            </td>
+          </tr>
+          <tr>
+            <td>Resized Image</td>
+            <td>
+              ![imagealttext](assets/simple-avatar.png =MxN)<br />
+              <em>M is width in pixels, N is height in pixels</em><br />
+              <em>constrain by ommitting one value, e.g.: =75x or =x75</em>
+            </td>
+            <td>
+              <Markdown>{avatarImageResized}</Markdown>
+              sample set @ 75x75
+            </td>
+          </tr>
+          <tr>
+            <td>Horizontal Line</td>
+            <td>--- <em>or</em> *** <em>or</em> ___</td>
+            <td>
+              <Markdown>---</Markdown>
+            </td>
+          </tr>
+          <tr>
+            <td>Tables</td>
+            <td colSpan="2"><a href="http://www.tablesgenerator.com/markdown_tables" rel="noopener noreferrer" target="_blank">use this website to generate markdown tables</a></td>
+          </tr>
+          <tr>
+            <td>URL Shortening</td>
+            <td colSpan="2">URLs can be written as /projects/username/projectname and /projects/username/projectname/pagename, omitting the &quot;https://www.zooniverse.org.&quot;</td>
+          </tr>
+        </tbody>
+      </table>
+      { talk &&
+        <TalkMarkdownHelp /> }
+      <p>If you need any more help formatting posts, please ask on the <a href="https://www.zooniverse.org/talk/">Zooniverse Talk</a> boards!</p>
+    </div>
+  );
+};
+
+
+MarkdownHelp.defaultProps = {
+  talk: false,
+  title: 'Guide to using Markdown'
+};
+
+MarkdownHelp.propTypes = {
+  talk: React.PropTypes.bool,
+  title: React.PropTypes.string
```

Comment:
For questions like this, you can look at the git blame information to find out who the original author was and ask the question. I think this might have been @mcbouslog's first work on PFE...